cryptoutil/digestset.go

```diff
@@ -190,16 +190,16 @@ func CalculateDigestSetFromFile(path string, hashes []crypto.Hash) (DigestSet, e
 	return CalculateDigestSet(file, hashes)
 }
 
-func (ds DigestSet) MarshalJSON() ([]byte, error) {
-	nameMap, err := ds.ToNameMap()
+func (first DigestSet) MarshalJSON() ([]byte, error) {
```

Comment:
Can you change this to `*DigestSet`?

Reply:
Marshal doesn't require a `pointer` receiver. https://pkg.go.dev/encoding/json@go1.19#RawMessage.MarshalJSON

I will revert the name change.

Reply:
👍